cmd/update-notifier.go

```diff
@@ -27,6 +27,58 @@ import (
 	"github.com/fatih/color"
 )
 
+// humanizeDuration converts time.Duration to human readable duration.
```

Comment:
The new PR is https://github.com/minio/minio/pull/4071

I have slightly changed humanization by having only higher component of time.Duration.  If its good to go, the package can't be used.